elastic/apm-server
# Add a _doc_count field for single-histogram metricsets

## Motivation/summary

If a metricset is published with a single histogram, add a `_doc_count` field which is the sum of the histogram's counts. This will enable us to remove some `value_count` aggregations in the UI to determine the transaction count.

## Checklist


## How to test these changes

1. Enable transaction metrics aggregation
2. Send some transactions
3. Check that the docs have `_doc_count` in them
4. Perform aggregations over the docs (e.g. terms agg on `transaction.name`), and check that the `doc_count` value in each bucket matches the number of transaction events aggregated.

## Related issues

Closes https://github.com/elastic/apm-server/issues/4474
